testnets/testnets.md

```diff
@@ -53,6 +73,28 @@ wasmcli query wasm list-code
 wasmcli keys add mywallet
 ```
 
+# Joining to be launched testnets
+
+You need to have your address and informations defined in networks genesis file to join for not yet launched testnets.
+Here is the script you can run to handle it automatically. It uses `coral` [network specific executables](https://github.com/CosmWasm/testnets/tree/master/coral#coral-wip):
+
+```sh
+cd $CW_DIR
+git clone git@github.com:CosmWasm/testnets
+cd testnets
+git checkout -b add-gen-acc-<validator-name>
+wasmd add-genesis-account $(wasmcli keys show -a mywallet) 100000000ushell,100000000ustake
+cat $HOME/.corald/config/genesis.json | jq '.app_state.auth.accounts [0]' > /tmp/new_acc.json
+NEW_GEN=$(jq '.app_state.auth.accounts += [input]' $CW_DIR/coral/genesis.json /tmp/new_acc.json) && echo $NEW_GEN > $CW_DIR/coral/genesis.json
+git add . && git commit -m "add <myvalidator account to coral genesis"
```

Comment:
With those changes you can remove the 2 lines above and jump directly to the git add